draft-ietf-jsonpath-base.md

```diff
@@ -902,11 +977,175 @@ size doesn't
 fit in the implementation's representation of an integer, the implementation
 MUST raise an error.
 
+### Descendant Selector
+
+#### Syntax
+{: unnumbered}
+
+The descendant selector starts with a double dot `..` and can be
+followed by an object member name (similar to the `dot-selector`),
+by an `index-selector` acting on objects or arrays, or by a wild card.
+
+~~~~ abnf
+descendant-selector = ".." ( dot-member-name      /  ; ..<name>
+                             index-selector       /  ; ..[<index>]
+                             index-wild-selector  /  ; ..[*]
+                             "*"                     ; ..*
+                           )
+~~~~
+
+#### Semantics
+{: unnumbered}
+
+The `descendant-selector` is inspired by ECMAScript for XML (E4X). It
+It selects the node and all its descendants..
```

Comment:
Repitition of "It" and extra period (ironically).

```suggestion
The `descendant-selector` is inspired by ECMAScript for XML (E4X).
It selects the node and all its descendants.
```

Reply:
Fixed.